src/main/java/fr/mrmicky/fastboard/FastBoardBase.java

```diff
@@ -363,12 +428,94 @@ public synchronized void updateLines(Collection<T> lines) {
                 if (!Objects.equals(getLineByScore(oldLines, i), getLineByScore(i))) {
                     sendLineChange(i);
                 }
+                if (!Objects.equals(getLineByScore(oldScores, i), getLineByScore(this.scores, i))) {
+                    sendScorePacket(i, ScoreboardAction.CHANGE);
+                }
             }
         } catch (Throwable t) {
             throw new RuntimeException("Unable to update scoreboard lines", t);
         }
     }
 
+    /**
+     * Update how a specified line's score is displayed on the scoreboard. A null value will reset the displayed
+     * text back to default. The scores will only be displayed for servers on 1.20.3 and higher.
+     *
+     * @param line the line number
+     * @param text the text to be displayed as the score. if null, no score will be displayed
+     * @throws IllegalArgumentException if the size of the texts does not match the current size of the board
+     * @throws IllegalStateException    if {@link #delete()} was call before
+     */
+    public synchronized void updateScore(int line, T text) {
+        checkLineNumber(line, true, false);
+
+        this.scores.set(line, text);
+
+        try {
+            if (customScoresSupported()) {
+                sendScorePacket(getScoreByLine(line), ScoreboardAction.CHANGE);
+            }
+        } catch (Throwable e) {
+            throw new RuntimeException("Unable to update line score", e);
+        }
+    }
+
+    /**
+     * Reset a line's score back to default (blank). The score will only be displayed for servers on 1.20.3 and higher.
+     *
+     * @param line the line number
+     * @throws IllegalArgumentException if the size of the texts does not match the current size of the board
+     * @throws IllegalStateException    if {@link #delete()} was call before
+     */
+    public synchronized void removeScore(int line) {
+        updateScore(line, emptyLine());
```

Comment:
```suggestion
        updateScore(line, null);
```
Technically, an empty score is not the same as a score with an empty text.